### PR TITLE
#4141 On focus in search box show history

### LINKED ISSUE #4141: On focus in search box we should suggest trending collections
On focus in searchbox we should suggest trending collections + search history, like last 3 searched terms 

@exezbcz can edit this to add Figma link but I guess it was in original one?

image

image

_Originally posted by @yangwao in https://github.com/kodadot/nft-gallery/issues/4127#issuecomment-1283751104_

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #4141
- [ ] Requires deployment <rubick/magick_issue>
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screen Shot 2022-10-25 at 8 47 13 PM

### CODE REVIEW

**Comment by yangwao** [comment]
Why I see it on bottom 👀

**Comment by Jarsen136** [comment]
> Why I see it on bottom 👀
> 
So do I. 
And the count of history result items is always one?

**Comment by exezbcz** [comment]
> > Why I see it on bottom 👀
> 
> So do I. And the count of history result items is always one?

No, it shouldn't be like that; I was probably thinking about three historical results.
Feedback @prachi00 
## card 
- text should be 16px
- cross icon is by default k-grey, on hover it changes color to black
- the whole card should have a hover effect similar to results cards, (hover bg in dark mode has same color as results card in dark mode) like this: 
image

Otherwise, good job!

**Comment by prachi00** [comment]
> * the whole card should have a hover effect similar to results cards, (hover bg in dark mode has same color as results card in dark mode) like this(figma):

I do see 3 though. do you always see one? 
Screen Shot 2022-10-26 at 6 48 19 PM

**Comment by prachi00** [comment]
> 

added these

**Comment by prachi00** [comment]
> > > Why I see it on bottom 👀
> > 
> > 
> > So do I. And the count of history result items is always one?
> 
> No, it shouldn't be like that; I was probably thinking about three historical results. Feedback @prachi00
> 
> ## card
> * text should be 16px
> * cross icon is by default k-grey, on hover it changes color to black
> * the whole card should have a hover effect similar to results cards, (hover bg in dark mode has same color as results card in dark mode) like this(figma):
>   image
> 
> ## others
> * History should not be displayed while you write something, thus it is only displayed with the trending collections popup. IMG:
>   image
> 
> Otherwise, good job!

added these

**Comment by Jarsen136** [comment]
> 
> I do see 3 though. do you always see one? Screen Shot 2022-10-26 at 6 48 19 PM

I see 3 items on RMRK now.
However, seems there is no history on BSX and also other chains
image

**Comment by prachi00** [comment]
> image At the bottom, there is no rating button, if you focus on search bar in navbar.

I dont understand, what rating button?

**Comment by yangwao** [comment]
there is typo in class 👀

File: styles/components/_search.scss [hunk 1]
```diff
@@ -203,6 +206,33 @@
...
+      font-size: 16px;
+    }
+
+    .times-icon {
+      &:hover {
+        color: $black;
+      }
+    }
+
+    .search-hisotry-item {
```
Comment (yangwao) [inline styles/components/_search.scss:229]: @prachi00 you can fix it in supplemental PR please 🙏

Comment (yangwao) [reply styles/components/_search.scss:229]: could be fixed in another PR I guess

**Review by yangwao — COMMENTED** [review]
just class typo )

**Comment by yangwao** [comment]
pay 80 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $80 @ 34.87 USD/KSM ~ 2.294 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0x8117c34389c18f7d5191386f0d8bc7e2b9524947308e1e960308639285a24730](https://kusama.subscan.io/extrinsic/0x8117c34389c18f7d5191386f0d8bc7e2b9524947308e1e960308639285a24730)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

File: styles/components/_search.scss (hunk 1 above)
Comment (prachi00) [reply styles/components/_search.scss:229]: I'll fix this yes

**Pull request merged**